spec/fixtures/auth_info_constants.rb

```diff
@@ -0,0 +1,103 @@
+module AuthInfoConstants
+  AUTH_URL = 'http://example.com/authorization'.freeze
+  TOKEN_URL = 'http://example.com/token'.freeze
+  REQUESTED_SCOPES = 'launch/patient openid fhirUser patient/*.*'.freeze
+  ENCRYPTION_ALGORITHM = 'ES384'.freeze
+  KID = '4b49a739d1eb115b3225f4cf9beb6d1b'.freeze
+  JWKS =
```

Comment:
Is there a reason not to just read the actual jwks file?

Reply:
nope! will update.